entries/lane/build.gradle

```diff
@@ -0,0 +1,14 @@
+apply plugin: 'java'
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile(project(':core'))
+    compile(project(':swing'))
+    compile(project(':entries:followsclosely'))
+    compile(project(':entries:jaron'))
```

Comment:
I think that these two lines shuld be removed. To compare agains other AIs the competition module should be used. As if we leave the dependencies in it will lead to circular dependencies.

Reply:
Makes sense - fixed